internal/elasticsearch/index/index.go

```diff
@@ -60,6 +60,7 @@ var (
 		"default_pipeline":                       schema.TypeString,
 		"final_pipeline":                         schema.TypeString,
 		"unassigned.node_left.delayed_timeout":   schema.TypeString,
+		"mapping.coerce":                         schema.TypeBool,
```

Comment:
This should be in `staticSettingsKeys`, this setting cannot be updated on an open index.